### PR TITLE
Albino Dino sprite/animation names

### PR DESCRIPTION
**Pull request by Ponmander**

### CODE REVIEW

File: tools/splat_ext/npc.yaml [hunk 1]
```diff
@@ -4933,49 +4933,49 @@ Duplighost:
...
+    - StillFwd
+    - StillBack
+    - Idle
+    - Walk
+    - Run
+    - Charge
+    - Pain
+    - Dead
+    - BurntPain
+    - BurntDead
```
Comment (HailSanta) [inline tools/splat_ext/npc.yaml:4975]: These two are usually BurnHurt and BurnStill

File: tools/splat_ext/npc.yaml [hunk 2]
```diff
@@ -4933,49 +4933,49 @@ Duplighost:
...
+    - Talk
+    - FwdTalk
```
Comment (HailSanta) [inline tools/splat_ext/npc.yaml:4977]: Suffix with Fwd/Back like you did for Still

File: tools/splat_ext/npc.yaml [hunk 3]
```diff
@@ -4933,49 +4933,49 @@ Duplighost:
...
-    - Anim0C
-    - Anim0D
+    - Still
+    - StillFwd
+    - StillBack
+    - Idle
+    - Walk
+    - Run
+    - Charge
+    - Pain
```
Comment (HailSanta) [inline tools/splat_ext/npc.yaml:4972]: There are usually Hurt/HurtStill

File: src/battle/area/pra/actor/albino_dino.c [hunk 4]
```diff
@@ -124,83 +124,83 @@ EvtScript N(handleEvent) = {
...
             EVT_RETURN
         EVT_CASE_EQ(EVENT_SPIN_SMASH_HIT)
             EVT_SET_CONST(LVar0, 1)
-            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_Anim08)
+            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_Pain)
             EVT_EXEC_WAIT(DoSpinSmashHit)
         EVT_CASE_EQ(EVENT_SPIN_SMASH_DEATH)
             EVT_SET_CONST(LVar0, 1)
-            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_Anim08)
+            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_Dead)
```
Comment (ethteck) [inline src/battle/area/pra/actor/albino_dino.c:150]: Some changes around here are wrong. 8 is pain, but you changed 8 to dead here. also the build doesn't match (jenkins failed, and your local build should be failing too) - please fix these

**Review by ethteck — CHANGES_REQUESTED** [review]
Jenkins is failing because the rom doesn't match anymore. I commented where the issue starts, I believe, but I think there are 2 or more mistakes. You can verify it's fixed yourself by running `ninja` and making sure you get a matching build.

**Comment by nanaian** [comment]
You can use `vbindiffvbindiff ver/us/baserom.z64 ver/us/build/papermario.z64` to view the differences:
image

**Comment by Ponmander** [comment]
I need to get back to this. I am hoping I just mixed up the animation IDs for things.

**Pull request merged**